docs/integrations/sources/microsoft-lists.md

```diff
@@ -0,0 +1,33 @@
+# Microsoft Lists
+Microsoft Lists connector enables seamless data integration and synchronization between Microsoft Lists and other destination. The connector leverages Microsoft Graph API to retrieve list items efficiently, ensuring smooth workflows and real-time data accessibility
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `client_id_2` | `string` | Client ID.  |  |
+| `client_secret_2` | `string` | Client secret.  |  |
+| `application_id_uri` | `string` | Application Id URI.  |  |
+| `site_id` | `string` | Site Id.  |  |
+| `tenant_id` | `string` | Tenant Id.  |  |
+| `domain` | `string` | Domain.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| lists | id | No pagination | ✅ |  ❌  |
+| listcontenttypes | id | No pagination | ✅ |  ❌  |
+| listitems |  | No pagination | ✅ |  ❌  |
+| items |  | No pagination | ✅ |  ❌  |
+| columnvalues | id | No pagination | ✅ |  ❌  |
```

Comment:
Without pagination it won't be possible to accept the connector. Please read more how Microsoft do pagination: https://learn.microsoft.com/en-us/graph/paging?tabs=http

Reply:
ya I also was going to add the pagination to this one(after entra id) tomorrow .

Reply:
@marcosmarxm I've modified the doc and added the pagination. Please check now and tell me if I've done anything wrong